.github/workflows/generate-component-labels.yml

```diff
@@ -0,0 +1,17 @@
+name: 'Generate component labels'
+on:
+  push:
+    branches: [main]
```

Comment:
We can add 
```yaml
    paths: [.github/CODEOWNERS]
```

here instead of using the git diff

Reply:
Very nice! I've updated it to use the CODEOWNERS file and the script to trigger the workflow.